src/core/languages/csharp.ts

```
@@ -0,0 +1,315 @@
+      const members: ASTSymbol[] = [];
+      const bodyNode = node.childForFieldName('body');
+      const basesNode = node.childForFieldName('bases');
+      const isAbstract = node.children.some(c => c.type === 'abstract');
+
+      if (bodyNode) {
+        for (const child of bodyNode.namedChildren) {
+          if (child.type === 'method_declaration') {
+            const method = extractMethod(child);
+            if (method) members.push(method);
+          } else if (child.type === 'constructor_declaration') {
+            const constructor = extractMethod(child);
```

Biome (2.1.2)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)